src/wix.vsconfig

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.7.2.TargetingPack"
```

Comment:
If we're doing this, should we add the `v141` toolset/libs too? I weep for what this does to the build time...

Reply:
Nope, not 0s. I had a cherry-pick failure that @rseanhall caught. It seems configuring VS takes 6+ min.